sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberClientBuilderTest.java

```diff
@@ -0,0 +1,499 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.administration;
+
+import com.azure.communication.administration.implementation.PhoneNumberAdminClientImpl;
+import com.azure.communication.common.CommunicationClientCredential;
+import com.azure.communication.common.HmacAuthenticationPolicy;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.policy.*;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.mockito.stubbing.Answer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@Execution(value = ExecutionMode.SAME_THREAD)
```

Comment:
Why is this required?

Reply:
It's needed for mocking in the unit tests. If the tests are run in parallel, the mocks will have conflicts.